seacrowd/sea_datasets/tydiqa/tydiqa.py

```diff
@@ -0,0 +1,422 @@
+    information-seeking task and avoid priming effects, questions are written by people who want to know the answer, but
+    don’t know the answer yet, (unlike SQuAD and its descendents) and the data is collected directly in each language
+    without the use of translation (unlike MLQA and XQuAD).
+    """
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(name="tydiqa_primary_task_source", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SOURCE_VERSION_P), schema="source",
+                       subset_id="tydiqa_primary_task"),
+        SEACrowdConfig(name="tydiqa_primary_task_indonesian_source", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SOURCE_VERSION_P), schema="source",
+                       subset_id="tydiqa_primary_task_indonesian"),
+        SEACrowdConfig(name="tydiqa_primary_task_thai_source", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SOURCE_VERSION_P), schema="source",
+                       subset_id="tydiqa_primary_task_thai"),
+        SEACrowdConfig(name="tydiqa_secondary_task_source", description=_SECONDARY_DESP,
+                       version=datasets.Version(_SOURCE_VERSION_S), schema="source",
+                       subset_id="tydiqa_secondary_task"),
+        SEACrowdConfig(name="tydiqa_secondary_task_indonesian_source", description=_SECONDARY_DESP,
+                       version=datasets.Version(_SOURCE_VERSION_S), schema="source",
+                       subset_id="tydiqa_secondary_task_indonesian"),
+        ##########################
+        SEACrowdConfig(name="tydiqa_primary_task_seacrowd_qa", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SEACROWD_VERSION), schema="seacrowd_qa",
+                       subset_id="tydiqa_primary_task"),
+        SEACrowdConfig(name="tydiqa_primary_task_indonesian_seacrowd_qa", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SEACROWD_VERSION), schema="seacrowd_qa",
+                       subset_id="tydiqa_primary_task_indonesian"),
+        SEACrowdConfig(name="tydiqa_primary_task_thai_seacrowd_qa", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SEACROWD_VERSION), schema="seacrowd_qa",
+                       subset_id="tydiqa_primary_task_thai"),
+        SEACrowdConfig(name="tydiqa_secondary_task_seacrowd_qa", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SEACROWD_VERSION), schema="seacrowd_qa",
+                       subset_id="tydiqa_secondary_task"),
+        SEACrowdConfig(name="tydiqa_secondary_task_indonesian_seacrowd_qa", description=_PRIMARY_DESP,
+                       version=datasets.Version(_SEACROWD_VERSION), schema="seacrowd_qa",
+                       subset_id="tydiqa_secondary_task_indonesian"),
+    ]
+    DEFAULT_CONFIG_NAME = "tydiqa_primary_task_source"
```

Comment:
Similar to the build_config comment, 
`DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_primary_task_source"`